Rakefile

```diff
@@ -1,21 +1,3 @@
-# Rakefile for testing rspec.
-#
-# October, 2013
-#
-# Copyright (c) 2013-2019 Cisco and/or its affiliates.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
```

Comment:
One of the sideeffects of performing a `pdk update` is that the licensing information is lost - currently we just add it to the rakefile, gemfile manually after an update

Reply:
Notice has been added back